_overviews/core/value-classes.md

```diff
@@ -56,6 +56,46 @@ The following fragment of `RichInt` shows how it extends `Int` to allow the expr
 At runtime, this expression `3.toHexString` is optimised to the equivalent of a method call on a static object
 (`RichInt$.MODULE$.extension$toHexString(3)`), rather than a method call on a newly instantiated object.
 
+### Location of extension methods
+
+There are several options regarding where to place the implemented extension methods.
+They can be placed in a package object which makes them available within the package automatically.
+
+    package object mypackage {
+      implicit class RichInt(val self: Int) extends AnyVal {
+        def toHexString: String = java.lang.Integer.toHexString(self)
+      }
+    }
+
+If the extension methods shall be reused it might be feasible to implement them not implicitly and include them via traits.
+
+    // in file: src/main/scala/mypackage/package.scala
+    package object mypackage extends mypackage.Shared {
+    }
+
+    // in file: src/main/scala/mypackage/Shared.scala
+    trait Shared {
+      implicit def toRichInt(self: Int): RichInt = new RichInt(self)
+    }
+
+    class RichInt(val self: Int) extends AnyVal {
+      def toHexString: String = java.lang.Integer.toHexString(self)
+    }
+
+However automatically available implicits have their drawbacks so it might be better to place them within a regular object.
+
+    object implicits {
```

Comment:
The convention is to use `syntax` here instead of `implicits`, I think.

Reply:
I've seen both but I think `syntax` might indeed make more sense here.

Reply:
> One thing I don't understand is why this is placed in the value-classes.md guide

agree, this would be much better placed in the style guide

Reply:
Well it started out as adding some stuff regarding value classes. ;-)